### DISCUSSION TITLE
Can't checkout to github generated branches like "3-api-|-create-rest-endpoint-to-forgot-password"

### DISCUSSION THREAD

**Discussion by Teddy-Fantahun** [community/community#102965] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

Github automatically creates branches from issues. But checking out to the branches locally result in the following error

error: pathspec '3-api-' did not match any file(s) known to git
bash: -create-rest-endpoint-to-forgot-password: command not found...

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Teddy-Fantahun ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
